msbuild/Xamarin.MacDev.Tasks.Core/Tasks/IBToolTaskBase.cs

```diff
@@ -132,14 +133,42 @@ IEnumerable<ITaskItem> GetCompilationDirectoryOutput (ITaskItem expected)
 			var extension = Path.GetExtension (expected.ItemSpec);
 			var dir = Path.GetDirectoryName (expected.ItemSpec);
 			var nibDir = expected.GetMetadata ("LogicalName");
+			var targets = GetTargetDevices (plist).ToList ();
```

Comment:
The directory should always exist, I added the check more for robustness than anything.